Automattic/wp-calypso
# TailoredFlows: skip domain selection using the same code path as othe…

#### Proposed Changes

* Make tailored flows follow the same code path, when the domain selection is skipped
* This fixes the issue where users get a different FREE domain than the one they selected


#### Testing Instructions
- Checkout this branch
- Test using the /start flow
- Select a free domain
- Check that the site created was with the same domain name as the one selected
- Test newsletter or LiB flow by skipping the domain selection and check for possible issues
